src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -0,0 +1,315 @@
+import React, { useState } from 'react';
+import { ArrayField, useFieldArray, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+import {
+  DatePeriod,
+  DatePeriodOptions,
+  FormWeekdays,
+  Language,
+  ResourceState,
+  ResourceStateApiOption,
+  ResourceStateOption,
+  TimeSpan as TimeSpanApiFormat,
+  TimeSpanFormFormat,
+} from '../../common/lib/types';
+import { transformToApiFormat } from '../../common/utils/date-time/format';
```

Comment:
It might be good idea to rename this to more specific, since it's applied outside of the file

Reply:
This is much older function than the PR, but I can rename this

Reply:
Oh good point. It seems it was applied in EditOpeningPeriodPage before, but not in CreateNewOpeningPeriodPage. Not directly in the scope of this PR, we can do it later

Reply:
I renamed the function as transformDateToApiFormat